ember-cli/2017-06/june-22.md

```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-06-22
+
+## Attendees
+
+
+## Agenda
```

Comment:
using [prettier](https://github.com/prettier/prettier) on the ember-cli codebase.